### ISSUE TITLE
Xanthoastrocytoma

### ISSUE THREAD

**Issue by DLewisSmith** [obophenotype/human-phenotype-ontology#6744]
**Preferred term label:**
Pleomorphic xanthoastrocytoma

**Synonyms**

**Definition (free text, please give PubMed ID)**
Pleomorphic xanthoastrocytomas are rare low-grade astrocytomas (WHO Grade II) typically found in the temporal lobe and classically presenting with epilepsy.

31535562
**Parent term (use hpo.jax.org/app)**
Astrocytoma HP:0009592

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-1735-8178

**Comment by pnrobinson** [comment]
added
